resources/aws_ec2.go

```diff
@@ -971,25 +1009,28 @@ func (obj *AwsEc2Res) snsListener(listenAddr string) (net.Listener, error) {
 }
 
 // snsPostHandler listens for posts on the SNS Endpoint.
-// TODO: download pem and check message against signature
 func (obj *AwsEc2Res) snsPostHandler(w http.ResponseWriter, req *http.Request) {
 	if req.Method != "POST" {
 		http.Error(w, "Invalid request method", http.StatusMethodNotAllowed)
 		return
 	}
-	// decode json
+	// Decode the post. If we are unable to decode the data, we print the
+	// error, ignore the post and return.
 	decoder := json.NewDecoder(req.Body)
 	var post postData
 	if err := decoder.Decode(&post); err != nil {
 		http.Error(w, "Bad request", http.StatusBadRequest)
-		select {
```

Comment:
let's keep this in but wrap it in a condition some sort of something ErrorXXXStrict